web/src/styles/base/global.scss

```diff
@@ -89,3 +89,14 @@ ol {
   background: linear-gradient(180deg, rgba(243, 239, 236, 0) 0%, #ecf3ec 100%);
   height: 80vh;
 }
+
+.screen-reader-only {
```

Comment:
Created this new class as "visibility-hidden" was altering the shape of the surrounding container. I'm open to renaming if it's not clear/ doesn't quite describe the behavior so much as the use case

Reply:
I think name is good.